ActionSheetPicker-3.0.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do | s |
     s.requires_arc = true
     s.ios.deployment_target = '6.1'
     s.platform = :ios
-    s.public_header_files = 'ActionSheetPicker.h', 'Pickers/*.h'
-    s.source_files = 'ActionSheetPicker.h', 'Pickers/*.{h,m}'
+    s.public_header_files = 'ActionSheetPicker.h', 'Pickers/{,Components/}*.h'
+    s.source_files = 'ActionSheetPicker.h', 'Pickers/{,Components/}*.{h,m}'
```

Comment:
Does it changes something?
I believe you have to add your picker to `ActionSheetPicker-3.0/CoreActionSheetPicker/CoreActionSheetPicker/CoreActionSheetPicker.h` instead

have a look: https://github.com/skywinder/ActionSheetPicker-3.0/blob/c78d7a8af26eb9e5f22ecb849c54804d6fd31be1/CoreActionSheetPicker/CoreActionSheetPicker/CoreActionSheetPicker.h